test/src/graphTest.cpp

```diff
@@ -220,6 +220,64 @@ TEST_F(Graph, FormatNodeResults)
 	}
 }
 
+TEST_F(Graph, UsePointsFromArray)
```

Comment:
Offtop:
`EXPECT_RGL_SUCCESS(rgl_graph_write_pcd_file(temporalMerge, "two_boxes_temporal_merge.pcd"));` in 164:
shouldnt be in `#ifdef RGL_BUILD_PCL_EXTENSION` ?